document_page_quality_manual/README.rst

```diff
@@ -1,39 +1,29 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
-    :alt: License
-
-==============
 Quality Manual
 ==============
 
-This module provides a quality manual template. The template has the same structure as the ISO 9001 standard.
+This module was written to provide a quality manual template. The template has the same structure as the ISO 9001 standard.
```

Comment:
IMO the original form is better: we describe what the module does, not how it was done.